R/fsi_functions.R

```diff
@@ -62,6 +89,45 @@ fsi_add_fsa <- function(fsi, lvar, tbl) {
   fsi
 }
 
+
+
+#' @title fsi_add_cs
+#' 
+#' @description fsi_add_cs adds the consequent to the FSI model
+#' 
+#' @usage 
+#' 
+#' fsi_add_cs(fsi, lvar, lvals, mfs, bounds)
+#' 
+#' @param fsi The FSI model instantiated with the `fsi_create` function.
+#' @param lvar A character value that represents a linguistic variable of the consequent.
+#' @param lvals A character vector that represents linguistic values of the linguistic variable of the consequent.
+#' @param mfs A character vector of corresponding membership functions.
```

Comment:
this is not a character vector, it is a vector of functions created by the `genmf` of the FuzzyR package.